tests/page-objects/common/DialogControl.ts

```diff
@@ -51,4 +54,25 @@ export default class DialogControl {
         );
         await dialogControlApplyBtn.click();
     }
+
+    async setRequiredCheckbox(value: boolean) {
+        await this.page.locator(DialogControlQa.requiredValueCheckbox).setChecked(value);
+    }
+
+    async checkCurrentValues(settings: {
+        fieldName: string;
+        appearance: {title: string};
+        defaultValue: string;
+        required: boolean;
+    }) {
+        expect(this.fieldName.getLocator().locator('input')).toHaveValue(settings.fieldName);
+        expect(this.appearanceTitle.textInput.getLocator().locator('input')).toHaveValue(
+            settings.appearance?.title,
+        );
+        expect(this.page.locator(slct(DialogControlQa.valueInput)).locator('input')).toHaveValue(
+            settings.defaultValue,
+        );
+
+        expect(await this.requiredCheckbox.getLocator().isChecked()).toEqual(settings.required);
```

Comment:
[await expect(locator).toBeChecked()](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-checked)